evalml/pipelines/components/estimators/classifiers/baseline_classifier.py

```diff
@@ -66,9 +67,9 @@ def predict(self, X):
         if strategy == "mode":
             return pd.Series([self._mode] * len(X))
         elif strategy == "random":
-            return self.random_state.choice(self._classes, len(X))
+            return get_random_state(self.random_state).choice(self._classes, len(X))
         else:
-            return self.random_state.choice(self._classes, len(X), p=self._percentage_freq)
+            return get_random_state(self.random_state).choice(self._classes, len(X), p=self._percentage_freq)
```

Comment:
Maybe I'm just super tired, but aren't we moving everything to get_random_seed?

Reply:
Answered below, but some scenarios require `np.random.RandomState` for the `choice()` method it has